packages/compass-collection/src/components/workspace/workspace.jsx

```diff
@@ -247,22 +186,13 @@ class Workspace extends PureComponent {
   render() {
     return (
       <div className={styles.workspace}>
-        <div className={styles['workspace-tabs']}>
-          <div onClick={this.props.prevTab} className={styles['workspace-tabs-prev']}>
-            <i className="fa fa-chevron-left" aria-hidden/>
-          </div>
-          <div className={styles['workspace-tabs-container']}>
-            {this.renderTabs()}
-            <CreateTab
-              createNewTab={this.onCreateNewTab}
-            />
-          </div>
-          <div className={styles['workspace-tabs-right']}>
-            <div onClick={this.props.nextTab} className={styles['workspace-tabs-next']}>
-              <i className="fa fa-chevron-right" aria-hidden/>
-            </div>
-          </div>
-        </div>
+        <WorkspaceTabs
```

Comment:
This may be a consequence of my lack of context here, but isn't there a bit of mix and match of responsibilities and knowledge split between this component and WorkspaceTabs right now, what is the idea behind both?

Reply:
What is the knowledge split here?
I think the idea is that `WorkspaceTabs` is a more straightforward rendering of the tabs for the workspace, and this `workspace.jsx` manages the rendering of the tabs and some of the global keyboard actions like creating a new tab with `ctrl/cmd + t`. If I'm not following your train of thought let's hop on a call?

Reply:
I think I'm probably just confused by the fact that there is a workspace package and then this file which seems to control `WorkspaceTabs` in every aspect through its own store. But that's just temporary or?

Reply:
Synced offline on how we can move towards a generic `compass-workspace` while addressing around the current collection centric logic in the `compass-collection` package. Down the line we'll use providers to supply the ipc handlers that are currently in this store. We can then manage listening to ipc events, like the menu actions from the main compass process as part of our react lifecycle. This'll probably clear up some bugs we currently have in Compass.